### ISSUE TITLE
Individual dates not selectable via keyboard

### ISSUE THREAD

**Issue by martygeek** [square/android-times-square#434]
I cannot select a data using the keyboard.  We have a requirement to support accessibility and when the calendar view open, the tab key only allows selection of the entire month.  There doesn't seem to be a way, via keyboard tab key and arrow keys, to navigate between days and select.   

Any ideas on how to make dates keyboard selectable?

**Comment by edenman** [comment]
@martygeek I'd be open to a PR but I don't have any ideas on how to go about it.  Probably best to start here: https://developer.android.com/training/keyboard-input/navigation.html

**Comment by martygeek** [comment]
It should be fairly simple stuff to add.   Accessibility for keyboard nav means the user should be able to direct focus to the elements on the screen using the tab or arrow keys.   In xml, you can set 'android"focusable="true" to a view and then the tab key on the keyboard will focus on that view.

**Comment by edenman** [comment]
Sounds good, go for it.

**Comment by martygeek** [comment]
since the calendar is a listview, adding that in XML might be a problem  You might need 'listView.setFocusableInTouchMode(true);'  or 'listView. setItemsCanFocus(true)'.    You might also have to explicitly set focus on the view in the listview with             android:descendantFocusability="afterDescendants"
and then listview.AddFocusables(ArrayList<View>, int direction, int focusableMode)
